### DISCUSSION TITLE
Labels on grid lines

### DISCUSSION THREAD

**Discussion by jidekrm** [imaNNeo/fl_chart#1569] [General]
How can I have labels on grid lines as shown in the attached image?. Best I have done so far is with FlTitlesData but it squeeze the texts in leftTitles  without using reservedSize.

![desired](https://github.com/imaNNeo/fl_chart/assets/35114550/efbf228d-de57-4020-ba3a-b46b75764a72)

**Comment by imaNNeo** [comment]
What if you use FlTitlesData and wrap your Text with [Transform.translate()?](https://api.flutter.dev/flutter/widgets/Transform/Transform.translate.html)

**Reply by jidekrm** [reply]
Thanks for your response.

Translating the label does work to a degree but without  SideTitles reservedSize, the labels needs a significant space to show up correctly but with the chart 'squeezed'.

First image here shows how it looks with not reservedSize set. Second one shows how it looks with reservedSize = 90.

Transform.translate Transform.scale on LineChart makes it fill up the space but so far not visually appealing.

<img width="356" alt="Screenshot 2024-02-08 at 12 35 42 PM" src="https://github.com/imaNNeo/fl_chart/assets/35114550/13a6c9e1-1265-40ef-a587-54ceb0346f5a">

<img width="348" alt="Screenshot 2024-02-08 at 12 36 29 PM" src="https://github.com/imaNNeo/fl_chart/assets/35114550/9a00d055-5dbe-4f4d-af52-47cd22ce6eec">
